### DISCUSSION TITLE
node ace list:routes error: Cannot find module

### DISCUSSION THREAD

**Discussion by galaczi** [adonisjs/core#3713] [Help] [answered]
I have updated my packages recently with ncu. I am not sure if that is the cause but when I run ``node ace list:routes`` I get the following error:

```
node ace list:routes

  Error 

 Cannot find module '.../node_modules/@adonisjs/core/build/commands/ListRoutes'


   1  resolveFileName
     .../node_modules/resolve-from/index.js:29

   2  resolveFrom
     /home/jeno/Dev/magantanarkereso/node_modules/resolve-from/index.js:43

   3  module.exports
     .../node_modules/resolve-from/index.js:46

   4  resolveFrom
     .../node_modules/@poppinss/utils/build/src/Helpers/resolveFrom.js:25

   5  ManifestLoader.loadCommand
     .../node_modules/@adonisjs/ace/build/src/Manifest/Loader.js:94
```
``node ace`` seems to be working fine otherwise.

I updated only minor versions of packages. Since then I deleted ``node_modules`` and installed again. 

I also tried with a fresh Adonisjs project, copied over ``package.json`` and ran ``npm install``. And I was able to list routes there. I must be having something in my project that causes this, but I am not familiar with what could affect the execution of commands. 

The ``node_modules/@adonisjs/core/build/commands/ListRoutes`` folder exists.

**Comment by Julien-R44** [comment, marked as answer]
Hello ! 
Take a look here : https://github.com/adonisjs/core/discussions/3671
Basically, just run `node ace generate:manifest`

**Reply by galaczi** [reply]
Indeed it solved it. Thank you!
